subjects/fromto/README.md

```diff
@@ -0,0 +1,45 @@
+## fromto
+
+### Instructions
+
+- Write a function that takes two `integers` and returns a string representing the range of numbers from the first to the second.
+- The function can return the numbers in any order.
+- The number must be separated by a comma and a space.
+- If any of the arguments is bigger than 99 or less than 0, the function returns `"invalid"` followed by a newline (`'\n'`).
```

Comment:
should be `invalid` instead of `"invalid"`. Should be (`\n`) instead of (`'\n'`).

Reply:
I was inspired by the old subjects. 
they do the same thing for the string `"` and char `'`
for example this subject of [`Displaya`](https://github.com/01-edu/public/tree/master/subjects/displaya)